go-controller/pkg/ovn/gateway_init.go

```diff
@@ -646,15 +680,15 @@ func findPolicyBasedRoutes(priority, compareTo string) ([]string, error) {
 	return strings.Split(policyIDs, "\n"), nil
 }
 
-func createPolicyBasedRoutes(match, priority, nexthops string) error {
+func createPolicyBasedRoutes(match, priority, nexthop string) error {
```

Comment:
yeah i initially changed this because id ont think you can specify more than one next hop, so nexthops doesnt make sense to me. But then I saw in the ovn schema it says nexthops...not sure why

Reply:
Because you can do ECMP on multiple nexthops with logical router policies now.

Reply:
hmm OK, I guess I haven't noticed that because of (and probably related to) https://bugzilla.redhat.com/show_bug.cgi?id=1939778

I'll revert this change